mesheryctl/internal/cli/root/system/status.go

```diff
@@ -178,7 +178,16 @@ var statusCmd = &cobra.Command{
 			// Print the data to a table for readability
 			utils.PrintToTable([]string{"Name", "Ready", "Status", "Restarts", "Age"}, data)
 
+			mesherySvc, err := utils.GetMesheryService(client, utils.MesheryNamespace)
+			if err != nil {
+				return err
+			}
 			log.Info("\nMeshery endpoint is " + mctlCfg.Contexts[mctlCfg.CurrentContext].Endpoint)
+			if len(mesherySvc.Status.LoadBalancer.Ingress) != 0 {
+				log.Info("\nLoadbalancer IP for Meshery: ", mesherySvc.Status.LoadBalancer.Ingress[0].IP)
```

Comment:
Do we always have one ingess in the Meshery? Is there not a case where this is more than one item?